sklearn/externals/setup.py

```diff
@@ -5,9 +5,5 @@ def configuration(parent_package='', top_path=None):
     from numpy.distutils.misc_util import Configuration
     config = Configuration('externals', parent_package, top_path)
     config.add_subpackage('joblib')
```

Comment:
is this still needed?

Reply:
Thanks for the review @NicolasHug !

Tried to remove it, but then unpickling of old pickles that we have in one test started to fail. So I suppose we still need it. Reverted that change.